scripts/pages/settings/mouse-sensitivity.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const CM_CONVERSION_FACTOR = 2.54;
+
+class MouseSensitivity {
+	static panels = {
+		/** @type {TextEntry} @static */
+		dpi: $('#DPI'),
+		/** @type {DropDown} @static */
+		unitSelection: $('#UnitSelection'),
+		/** @type {TextEntry} @static */
+		unitsPer360: $('#UnitsPer360')
+	}
+	
+	static calculateSensitivity(){
+		if(!this.panels.dpi.text) return;
+		
+		const yaw = GameInterfaceAPI.GetSettingFloat('m_yaw')
+		const eDPI = this.panels.dpi.text * GameInterfaceAPI.GetSettingFloat('sensitivity');
+		let per360 = (360 / (eDPI * yaw));
+
+		if(this.panels.unitSelection.GetSelected().id === 'm_sens_cm')
+		per360 *= CM_CONVERSION_FACTOR;
+		
+		//displayprecision attribute does not work on TextEntry
+		per360 = Math.round(per360 * 100)/100;
+		this.panels.unitsPer360.text = per360;
```

Comment:
Bit cleaner to use `toFixed` here, so `this.panels.unitsPer360.text = per360.toFixed(2);`.